### PR TITLE
installation medias creation on MacOS: 32MB -> 32M

### PR DESCRIPTION
**Pull request by caldav**
https://bugs.launchpad.net/developer-ubuntu-com/+bug/1703780

### CODE REVIEW

File: templates/pages/core/get-started/installation-medias.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Terminal app there), then run the following command:
 
  * You can now copy the image to the SD card, using the following command:
 
-        sudo dd if=~/Downloads/<image file> of=<drive address> bs=32MB
+        sudo dd if=~/Downloads/<image file> of=<drive address> bs=32M
```
Comment (ajabble) [inline templates/pages/core/get-started/installation-medias.md]: I am still getting same error with above fix, but `bs=32m` ('m' in lowercase) works for me.

**Comment by anthonydillon** [comment]
@ajabble does this solve the issue?

**Comment by ashish-jabble** [comment]
@anthonydillon yupe, go ahead with the change.

**Review by anthonydillon — APPROVED** [review]
Thanks.

**Pull request merged**